lib/rubocop/cop/rspec/implicit_subject.rb

```diff
@@ -42,21 +42,36 @@ module RSpec
       #   # good
       #   it { expect(subject).to be_truthy }
       #
+      # @example `EnforcedStyle: always`
+      #   # bad
+      #   it { expect(subject).to be_truthy }
+      #
+      #   # good
+      #   it { is_expected.to be_truthy }
+      #
       class ImplicitSubject < Base
         extend AutoCorrector
         include ConfigurableEnforcedStyle
 
-        MSG = "Don't use implicit subject."
-        RESTRICT_ON_SEND = %i[is_expected should should_not].freeze
+        RESTRICT_ON_SEND = %i[
+          expect
+          is_expected
+          should
+          should_not
+        ].freeze
+
+        # @!method explicit_subject?(node)
+        def_node_matcher :explicit_subject?, <<-PATTERN
```

Comment:
Note that this way we won't flag usages of a named subject, e.g. code like
```
subject(:instance) { described_class.new }

it { expect(instance).to be_valid }
```

will still be allowed